### PR TITLE
feature-modal

### PR DESCRIPTION
**Pull request by Bhakti199**
here I added standard modal.

### CODE REVIEW

**Review by munjalSaksham71 — APPROVED** [review]

File: modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  color: var(--dark-grey);
+}
+.modal-button {
+  display: flex;
+  justify-content: flex-end;
+  gap: 10px;
+}
+/* iframe */
+iframe {
+  width: 37rem;
```
Comment (tusharkandpal) [inline modal/modal.css:31]: Giving fix values may cause problem in responsive design of your library

File: modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+@import url(/style.css);
+@import url(/button/button.css);
+
+.modal-container {
+  margin-top: 2rem;
```
Comment (tusharkandpal) [inline modal/modal.css]: Can combine these two lines in one. `margin: 2rem 0;`

File: modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>RangerUI | docs</title>
+  </head>
+
+  <body>
+    <nav class="navbar">
+      <span class="navbar-heading"
+        ><a href="" class="ranger-ui ranger-ui-name">RangerUI</a></span
```
Comment (tusharkandpal) [inline modal/modal.html]: `href` attribute is empty

**Review by shaksham08 — APPROVED** [review]

**Review by shaksham08 — APPROVED** [review]
Looks Good

**Review by tusharkandpal — CHANGES_REQUESTED** [review]

**Review by tusharkandpal — APPROVED** [review]
Looks good to merge now. Great work with library! 👍🙂

**Pull request merged**